venues/ICLR.cc/2019/Conference/webfield/reviewerWebfield.js

```diff
@@ -58,12 +58,17 @@ var getPaperNumbersfromGroups = function(groups) {
 };
 
 var getBlindedNotes = function(noteNumbers) {
-  var noteNumbersStr = noteNumbers.join(',');
+  if (noteNumbers.length) {
+    var noteNumbersStr = noteNumbers.join(',');
+
+    return $.getJSON('notes', { invitation: CONFERENCE + '/-/Blind_Submission', number: noteNumbersStr, noDetails: true })
+      .then(function(result) {
+        return result.notes;
+      });
+  } else {
+    return $.Deferred().resolve([]);
```

Comment:
In general I prefer the style of "returning early" meaning that instead of doing `if (noteNumbers.length)` you would do `if (!noteNumbers.length) return ...` and then leave the rest of the function the same. This has the advantage of reducing the indentation level, making the code easier to read, and reducing the size of the diffs.